org.ampr.ct1enq.tags.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id" : "org.ampr.ct1enq.tags",
```

Comment:
Hi @bbhtt the ampr.org is a domain reserved for radio amateurs and is self administered by "hams". I've several domains in the ampr.org domain since the nineties, assigned to me via the regional administrators. It's used for IP over the air in the 44.x.x.x range. 

You can read more about the *am*ateur *p*acket *r*adio  network (ampr.org) [here](https://en.wikipedia.org/wiki/AMPRNet).

I must own/control the domain? 

Addresses in the ampr.org have a unique identifier which is the government issued callsign, mine is ct1enq.

Isn't this valid?

Reply:
> I must own/control the domain?

Yes, and reachable over the internet

Reply:
Ok, I'll try to setup a github.io page. :/

Reply:
So that means that i can use io.github.phastmike.tags (e.g.) without setting up the domain?

Reply:
Assuming I change the appid, the branch does not need to, and cannot, be changed but the manifest should be deleted and re-added with the new appid, right?

Reply:
> So that means that i can use io/com.github.phastmike.tags (e.g.) without setting up the sub/domain? 

Yes, use `io.github.{project}`

Reply:
> Assuming I change the appid, the branch does not need to, and cannot, be changed but the manifest should be deleted and re-added with the new appid, right?
> 
> now: org.ampr.ct1enq.tags.json
> new: io.github.phastmike.tags.json

The branch can stay the same, but yes the manifest name needs to change along with some of the contents like id/app-id, file names etc.

Reply:
Also make sure to do all the upstream code/file name changes too